components/presence/presence.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div
+    v-if="presence"
+    :class="['d-presence', BACKGROUND_COLOR_FOR_THEME[theme]]"
+    data-qa="dt-presence"
+  >
+    <span
+      v-if="srText"
+      data-qa="dt-presence-sr-text"
+      class="sr-only"
+    >{{ srText }} </span>
```

Comment:
Should this optionally be aria-live="polite" in case we want to live announce a change in status? Not sure the needs of product, maybe it's not necessary.

Reply:
Good point, I missed this but my thinking originally to add the `role=status` attribute to the root element, which will implicitly add `aria-live: polite` to its child elements: https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA22

Reply:
Sounds good but we should probably also include a way to disable this. I'm thinking if you have 100 contacts in your contacts list with a presence indicator and every time their presence updates it notifies you it would be really annoying.

Reply:
I added a way to set `aria-live` directly on the element by parent. 

```vue
  <div
    class="d-presence"
    data-qa="dt-presence"
    role="status"
    :aria-live="$attrs.ariaLive || 'polite'"

```

Reply:
Hmm the more I think about this the more I think it should just be off by default. In which case we actually don't need to set this property at all since it can be set via attrs. Lets remove it and just mention that you can optionally do it in the accessibility section of the docs. Sorry for the confusion.

Reply:
Set it to default to 'off', updated tests & docs.